jdatetime/__init__.py

```diff
@@ -541,7 +547,7 @@ def strftime(self, format):
 
         if '%c' in format:
             format = format.replace(
-                "%c", self.strftime("%a %b %d %H:%M:%S %Y"))
+                "%c", self.strftime("%a %d %b %y, %H:%M:%S"))
```

Comment:
if I check `%c` format in `datetime`:
```
datetime.datetime.now().strftime("%c")
'Tue Aug 20 09:31:37 2019'

datetime.datetime.now().strftime("%a %b %d %H:%M:%S %Y") == datetime.datetime.now().strftime("%c")
True

datetime.datetime.now().strftime("%a %d %b %y, %H:%M:%S") == datetime.datetime.now().strftime("%c")
False
```

Is there any reason that `%c` has changed?